tests/kafkatest/version.py

```diff
@@ -63,8 +63,13 @@ def reassign_partitions_command_supports_bootstrap_server(self):
         return self >= V_2_5_0
 
     def kafka_configs_command_uses_bootstrap_server(self):
+        # everything except User SCRAM Credentials (KIP-554)
         return self >= V_2_6_0
 
+    def kafka_configs_command_uses_bootstrap_server_scram(self):
+        # User SCRAM Credentials (KIP-554)
+        return self > LATEST_2_6
```

Comment:
I don't think this is quite right.  It should be `self >= V_2_7_0`, right?  And we should add 2.7 to the `version.py` file.  

master branch is already identifying itself as 2.7 anyway.

```
[cmccabe@zeratul kafka2]$ git grep 2.7.0 | tail -n 10
docs/upgrade.html:<h5><a id="upgrade_270_notable" href="#upgrade_270_notable">Notable changes in 2.7.0</a></h5>
gradle.properties:version=2.7.0-SNAPSHOT
kafka-merge-pr.py:DEFAULT_FIX_VERSION = os.environ.get("DEFAULT_FIX_VERSION", "2.7.0")
streams/quickstart/java/pom.xml:        <version>2.7.0-SNAPSHOT</version>
streams/quickstart/java/src/main/resources/archetype-resources/pom.xml:        <kafka.version>2.7.0-SNAPSHOT</kafka.version>
streams/quickstart/pom.xml:    <version>2.7.0-SNAPSHOT</version>
streams/src/main/java/org/apache/kafka/streams/StreamsBuilder.java:     * @deprecated Since 2.7.0; use {@link #addGlobalStore(StoreBuilder, String, Consumed, ProcessorSupplier)} instead.
streams/streams-scala/src/main/scala/org/apache/kafka/streams/scala/StreamsBuilder.scala:    "2.7.0"
tests/kafkatest/__init__.py:__version__ = '2.7.0.dev0'
tests/kafkatest/version.py:DEV_VERSION = KafkaVersion("2.7.0-SNAPSHOT")
```

Similarly `supports_tls_to_zookeeper` should be checking for 2.5 and later, not "after 2.4" (they should be the same, but what if someone messed up incrementing LATEST_2_4, etc....)  It's always better to be explicit about what version something appeared in than to say it happened "after (but not including) some version X"